cli/exchange/node.go

```diff
@@ -173,10 +173,17 @@ func NodeUpdate(org string, credToUse string, node string, filePath string) {
 			verifyNodeUserInput(org, credToUse, node, nId, ui)
 			patch[k] = ui
 
+		} else if k == "nodeType" {
```

Comment:
This should not be allowed.

Reply:
why? user should be allowed to change the node type if the node is not registered.

Reply:
removed